### ISSUE TITLE
Some users cannot configure stylelint correctly

### ISSUE THREAD

**Issue by budarin** [shinnn/vscode-stylelint#224]
VS Code: 1.24.1
.vscode/settings: 
```js
    "stylelint.enable": true,
    "css.validate": false,
    "less.validate": false,
    "scss.validate": false
```
vscode-stylelint: 0.37.0
stylelint: 9.3.0
stylelint config: "stylelint.config.js"
Repository: [link](https://github.com/budarin/react-bootstrap-project)

**Comment by shinnn** [comment]
Use [Stack Overflow](https://stackoverflow.com/) for support questions. Whether your stylelint config is valid or invalid is not something to do with this repository.
